R/tt_toString.R

```diff
@@ -435,7 +435,6 @@ get_formatted_fnotes <- function(tt) {
   )
 
   inds <- vapply(lst, ref_index, 1L)
-  stopifnot(all(is.na(inds)) || !is.unsorted(inds))
```

Comment:
why is this no more valid?

Reply:
It's not required that indices are sorted after this change - for example, if you have footnotes 1, 2, and 3, and footnote 1 is repeated in a later cell, you might have indices ordered [1, 2, 3, 1], which fails this check.

Reply:
To clarify, I left all footnotes as separate entries in `ref_fnote_df` from `matrix_form` and just changed the symbol/ref_index columns. The symbol info seems to be arbitrary as I can't find anywhere that it's used. Then in formatters, I removed the auto-reindexing (`fix_fnote_df` which overrides indices to 1, 2, 3, ...), and set the `toString` method to only print each unique ref footnote once.